aarch64_extension.cpp

```diff
@@ -124,6 +210,42 @@ class AArch64ArchitectureExtension : public ArchitectureHook {
             {{"reg", Type::IntegerType(8, true, ""), true, Variable()},
              {"val", Type::IntegerType(8, true, ""), true, Variable()}},
             false, 0));
+
+    std::vector<uint32_t> registers = this->m_base->GetAllRegisters();
+    uint32_t reg = 0;
+    if (!registers.empty()) {
+      reg = 1 + *max_element(registers.begin(), registers.end());
+    }
+
+    for (size_t Vd = 0; Vd < 32; Vd++) {
+      char baseRegisterName[3];
+
+      snprintf(baseRegisterName, sizeof(baseRegisterName), "v%zu", Vd);
+
+      for (auto& form : regForms) {
+        BNRegisterInfo regInfo;
+        char* regNameCStr = NULL;
+
+        if (asprintf(&regNameCStr, "v%zu%c[%zu]", Vd, form.elemType,
+                     form.vectorIndex) < 0) {
+          LogError("asprintf: %s", strerror(errno));
+          /* XXX: should this be a C++ exception or something? */
+          assert(false);
```

Comment:
See note above

Reply:
(I meant to reference https://github.com/meme/aarch64-extensions/pull/7#discussion_r414068234)